src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -873,7 +873,7 @@ internal sealed partial class LazyBinaryOperatorExpression : BaseBinaryOperatorE
     internal abstract class BaseTupleBinaryOperatorExpression : Operation, ITupleBinaryOperation
     {
         public BaseTupleBinaryOperatorExpression(BinaryOperatorKind operatorKind, SemanticModel semanticModel, SyntaxNode syntax, ITypeSymbol type, Optional<object> constantValue, bool isImplicit)
-            : base(OperationKind.BinaryOperator, semanticModel, syntax, type, constantValue, isImplicit)
+            : base(OperationKind.TupleBinaryOperator, semanticModel, syntax, type, constantValue, isImplicit)
```

Comment:
Seems like an unrelated bug fix, can we break it out into a separate change?

Reply:
Ah, nevermind, I see that these were in a separate commit. To satisfy my curiosity, why were these bug fixes made in a separate branch, and not in master directly?

---
In reply to: [188370704](https://github.com/dotnet/roslyn/pull/26862#discussion_r188370704) [](ancestors = 188370704)

Reply:
I actually thought they'd take longer and be more disruptive. Hence we did them in a branch instead. Turns out it was much easier than I thought based on the number of failures we started with.